[docs] CDC: Update content for CDC streams and transactional ordering (#25112)
yugabyte/yugabyte-db: docs/content/preview/explore/change-data-capture/using-yugabytedb-grpc-replication/debezium-connector-yugabytedb.md

| tombstones.on.delete | `true` | Controls whether a delete event is followed by a tombstone event.<br/><br/> `true` - a delete operation is represented by a delete event and a subsequent tombstone event.<br/><br/> `false` - only a delete event is emitted.<br/><br/> After a source record is deleted, emitting a tombstone event (the default behavior) allows Kafka to completely delete all events that pertain to the key of the deleted row in case log compaction is enabled for the topic. |
| auto.add.new.tables | `true` | Controls whether the connector should keep polling the server to check if any new table has been added to the configured change data stream ID. If a new table has been found in the stream ID and if it has been included in the `table.include.list`, the connector will be restarted automatically. |
| new.table.poll.interval.ms | 300000 | The interval at which the poller thread will poll the server to check if there are any new tables in the configured change data stream ID. |

> | transaction.ordering | `false` | Whether to order transactions by their commit time.<br/><br/>{{< note title="Note" >}} This configuration property has been deprecated, see [transaction ordering](#transaction-ordering) for more details. {{< /note >}} |

Reviewer: Should we use a warning here? @aishwarya24, and is this how we send a deprecation notice if something is part of the configuration table?

Reviewer 2: I have made this a warning.

## Transaction ordering

{{< warning title="Deprecation Notice" >}}

> Starting with YugabyteDB v2024.2, and YugabyteDB gRPC Connector `dz.1.9.5.yb.grpc.2024.2`, the configuration `transaction.ordering` is deprecated. This configuration will be removed in future releases. As the [YugabyteDB connector](../using-logical-replication/yugabytedb-connector) based on logical replication model offers the same transactional ordering properties by default, you are advised to use the same for your use cases.

Reviewer 3: ```suggestion
Starting with YugabyteDB v2024.2, and YugabyteDB gRPC Connector `dz.1.9.5.yb.grpc.2024.2`, the configuration `transaction.ordering` is deprecated. This configuration will be removed in future releases. As the PostgreSQL Logical Replication-based [YugabyteDB connector](../../using-logical-replication/yugabytedb-connector) offers the same transactional ordering properties by default, you are advised to use the same for your use cases.
```

> Currently, for cases where using transactional ordering is absolutely necessary with the gRPC connector, add the following configurations to use the deprecated configuration:

Reviewer 3: ```suggestion
Currently, for cases where you absolutely need to use transactional ordering with the YugabyteDB gRPC Connector, add the following configurations to use the deprecated configuration:
```

```output.json
{
  ...
  "transaction.ordering":"true",
  "TEST.override.transaction.ordering.deprecation":"true"
  ...
}
```

{{ < /note > }}

> 

Reviewer: This is also not rendering.
